templates/people.html

```diff
@@ -47,7 +47,7 @@ <h3 class="site-section-meta">{{ person.name }}</h3>
         <h4 class="font-italic">People</h4>
         <ul>
           {% for person in people.current_members %}
-            <li><a href="#{{ person.name | replace(' ','_') }}">{{ person.name }}</a></li>
+            <li><a href="#{{ person.tag_id }}">{{ person.name }}</a></li>
```

Comment:
What is the purpose of this change?

Reply:
It makes the link more descriptive, and now that I look at it, looks better than the custom template tags. Plus we can change it later once in the function in `main.py` instead of changing it multiple times in the template.

Reply:
This is an internal tag to the profile of someone in the lab (not to a photo). `photo_of_X` seems less clear than using the name.